mappings/existing_model_link_translation.go

```diff
@@ -0,0 +1,22 @@
+package mappings
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+)
+
+//go:embed translation/existing_model_link.json
+var ExistingModelLinkTranslationJSON []byte
+
+func ExistingModelLinkTranslation() (*model.ExistingModelLinkTranslation, error) {
+	var translation model.ExistingModelLinkTranslation
+	err := json.Unmarshal(ExistingModelLinkTranslationJSON, &translation)
+	if err != nil {
+		fmt.Println("Error unmarshalling JSON:", err)
+		return nil, err
+	}
+	return &translation, nil
```

Comment:
I wonder if this can be abstracted into a helper or base method which takes the translation as an argument. It seems like very little of this code is specific to the implementation. The only meaningfully distinct element appears to be the go:embed line.

Reply:
Yeah that's true, I don't think I will try to do this in this pass. I don't think this current implemention hurts, but we could change it in the future if desired.